### PR TITLE
Add timestamp to catch timeout error on metal3-dev-env integration test

### PR DESCRIPTION
**Pull request by fmuyassarov**
This patch tracks time of a single pipeline stage in order to kill the integration job if timeout happens during the test.

### CODE REVIEW

File: ci/jobs/capi_bm_integration_tests.pipeline [hunk 1]
```diff
@@ -84,6 +92,13 @@ pipeline {
   }
   post {
     cleanup {
+      script {
+        CURRENT_BUILD_TIME = System.currentTimeMillis()
+        if ((((CURRENT_BUILD_TIME - CURRENT_START_TIME)/1000) - 3600) > 0) {
```
Comment (maelk) [inline ci/jobs/capi_bm_integration_tests.pipeline]: Can you parameterize the timeout duration like TIMEOUT*60 and use TIMEOUT variable in the stage ?

Comment (fmuyassarov) [reply ci/jobs/capi_bm_integration_tests.pipeline]: done

File: ci/jobs/capi_bm_integration_tests.pipeline [hunk 2]
```diff
@@ -37,6 +38,7 @@ pipeline {
     BUILD_URL = "${env.BUILD_URL}"
     PR_ID = "${ghprbPullId}"
     DISTRIBUTION = "${DISTRIBUTION}"
+    TIMEOUT = 3600
```
Comment (maelk) [inline ci/jobs/capi_bm_integration_tests.pipeline]: Does this need to be defined twice ?

Comment (fmuyassarov) [reply ci/jobs/capi_bm_integration_tests.pipeline]: No. Removed

**Pull request merged**